exotic/inputs.py

```diff
@@ -493,11 +493,11 @@ def target_star_coords(coords, planet):
     if isinstance(coords, list) and len(coords) == 2:
         pass
     elif isinstance(coords, str) and any(str.isdigit(x) for x in coords):
-        coords = ' '.join(re.sub('\D', ' ', coords).split()).split(' ')
-        coords = [int(i) for i in coords]
+        coords = re.findall(r"[-+]?(?:\d*\.*\d+)", coords)
+        coords = list(map(lambda coord: int(float(coord)), coords))
```

Comment:
- Could `int(float(coord))` be simply `int(coord)`? 
- Instead of invoking map with an anonymous lambda could this be done more directly with just a different comprehension?

Reply:
I do `int(float(coord))` in case the user enters a float as coordinate values rather than integers. It avoids a `ValueError` for conversion:

> ValueError: invalid literal for int() with base10: '865.01'

Reply:
That's possible, I'll change it to the following:

> [int(float(coord)) for coord in coords]

Reply:
I see why now. Resolved. Thanks.

Reply:
That's correct. Thanks for clearing that up. Resolved.